samples/matter/lock/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -22,4 +22,4 @@ CONFIG_CHIP_FACTORY_DATA=n
 CONFIG_CHIP_FACTORY_DATA_WRITE_PROTECT=n
 
 # Set SUIT template paths
-CONFIG_SUIT_ENVELOPE_TEMPLATE="suit_templates/app_envelope.yaml.jinja2"
+CONFIG_SUIT_ENVELOPE_TEMPLATE="${ZEPHYR_NRF_MODULE_DIR}../common/suit_templates/app_envelope.yaml.jinja2"
```

Comment:
bit confused by this, if this is reading in a file, why is there a "common" folder in the same location as `nrf`, if this is writing a file then why is it being written to here?

Reply:
This is a base SUIT manifest that we suggest to customers, but they can have their own manifests and use the `CONFIG_SUIT_ENVELOPE_TEMPLATE` config to set the new path. Do you recommend setting it by default within the sample's Kconfig file?

Reply:
To explain it more, actually, we will have one "base" SUIT manifest template that is valid for all Matter samples, and it is located in the `nrf/samples/matter/common` directory. We need to set the `CONFIG_SUIT_ENVELOPE_TEMPLATE` config anyway to provide a template manifest path so we decided to provide a path to the common directory instead of having the copy of the manifests in each app separately. Users can write their own manifest templates, and then they can easily set the `CONFIG_SUIT_ENVELOPE_TEMPLATE` config to other paths and use them. They can both modify `samples/matter/lock/boards/nrf54h20dk_nrf54h20_cpuapp.conf` file or provide the config as CMAKE/NINJA build argument as well.

Reply:
I would really appreciate it if you didn't rush to get things merged that clearly are wrong. To highlight:
```
message(WARNING "path: ${ZEPHYR_NRF_MODULE_DIR}../common/suit_templates/app_envelope.yaml.jinja2")
```
=
`  path: /tmp/bb/nrf../common/suit_templates/app_envelope.yaml.jinja2`
Notice something odd about the path? Please fix

Reply:
3 our tasks were blocked because we've been waiting to merge this PR for two weeks... I explained why is wrong and I hadn't got any input from you for two days, so to unlock the other tasks, we merged this one, and as I wrote below, I can do a follow-up if this is wrong. I didn't provide any regression, because it works for me, so I still don't understand why your message points to the wrong path, whereas SUIT generation works properly...